src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -154,10 +154,10 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'MonoAndroid7'">
     <PackageReference Include="Xamarin.Android.Support.CustomTabs">
-      <Version>25.3.1</Version>
+      <Version>25.4.0.2</Version>
```

Comment:
>25.4.0.2 [](start = 15, length = 8)

Ideally we would have a script which updates this one automatically based the right parameters such as MonoAndroid version etc. I was looking for information on this (and how to get the right version of Java installed) however I didn't find this info handy. If we have the info perhaps we can make it more visible? Or get something added to our readme file.